Lib/test/test_enum.py

```diff
@@ -5221,7 +5239,33 @@ def __new__(cls, value, label):
             VEST = 1, 'uppert half'
             PANTS = 2, 'lower half'
         _test_simple_enum(CheckedComplexFlag, ComplexFlag)
-
+        #
+        #
+        class HasColor(ABC):
+            @abstractmethod
+            def color(self):
+                ...
+        class CheckedABCMixin(HasColor, Enum):
+            ROSES = 1
+            VIOLETS = 2
+            def color(self):
+                match self:
+                    case self.ROSES:
+                        return 'red'
+                    case self.VIOLETS:
+                        return 'blue'
+        #
+        @_simple_enum(Enum)
+        class ABCMixin(HasColor):
+            ROSES = 1
+            VIOLETS = 2
+            def color(self):
+                match self:
+                    case self.ROSES:
+                        return 'red'
+                    case self.VIOLETS:
+                        return 'blue'
+        _test_simple_enum(CheckedABCMixin, ABCMixin)
 
 class MiscTestCase(unittest.TestCase):
```

Comment:
Could you revert the newline you removed?

```suggestion
        _test_simple_enum(CheckedABCMixin, ABCMixin)


class MiscTestCase(unittest.TestCase):
```

Reply:
You can resolve this, and to apply a suggestion you can click commit suggestion.